ByeBoo-iOS/fastlane/Fastfile

```
@@ -18,15 +18,103 @@ default_platform(:ios)
+      upload_to_app_store(
+        skip_screenshots: true,
+        skip_metadata: true,
+        force: true,
+        submit_for_review: true,               # 자동으로 심사 제출 
+        automatic_release: true,               # 심사 통과 후 자동 출시
+        precheck_include_in_app_purchases: false,
+        submission_information: {
+          add_id_info_uses_idfa: false,          # admob 추가 시 true로 변경 
+          export_compliance_uses_encryption: false  
+        }
+    rescue => e
```

RuboCop (1.88.2)

[fatal] unexpected token kRESCUE
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)